script.js

```diff
@@ -13,5 +15,55 @@ hamburger.addEventListener('click', () => {
 document.querySelectorAll('.nav-link').forEach((n) => n.addEventListener('click', () => {
   hamburger.classList.remove('active');
   navMenu.classList.remove('active');
-  logo.classList.toggle('active');
-}));
\ No newline at end of file
+  logo.classList.remove('active');
+}));
+
+project.addEventListener('click', () => {
+  const main = document.createElement('div');
+  main.className = 'main-popup';
+  const popup = document.createElement('div');
+  popup.className = 'popup';
+  popup.innerHTML = `
+<div class="popup-menu">
+<div class="popup-header">
+<h2 class="popup-title">Multi Post Stories</h2>
+<h2 class="popup-title-2">Keeping track of hundreds  of components website</h2>
+<button class="close-button">&times;</button>
+</div>
+<div class="popup-languages">
+<ul class="languages-work">
+                <li class=" popup-lan-box"><p class="code-name pop-name">html</li>
+                <li class=" popup-lan-box"><p class="code-name pop-name">bootstrap</p></li>
+                <li class=" popup-lan-box"><p class="code-name pop-name">Ruby</p></li>
+              </ul>
+   </div>
+   <div class ="desktop-popup">
+   <img class='popup-image' src="/img/Snapshoot Portfolio.png" alt="popup work preview">
+   <div class="popup-end">
+   <p class="lorem-1">Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s,
+    when an unknown printer took a galley of type and scrambled it 1960s with the releaLorem Ipsum is simply dummy text of the printing and typesetting industry.
+     Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, 
+   when an unknown printer took a galley of type and scrambled it 1960s with the releax map lapora verita.</p>
+   <p class="lorem">Lorem Ipsum is simply dummy text of the printing and typesetting industry. 
+   Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries,
+    but also the leap into electronic typesetting, remaining essent</p>
+    <div class="popuplinks"><a href="https://cluab.github.io/Portfolio/">
+    <button class="live">See Live<img src="/img/Icon.png" alt="icon-image"></button></a>
+    <a href="https://github.com/Cluab/Portfolio">
+    <button class="live">See Source<img src="/img/Vector.png" alt="github-icon"></button></a></div>
+    </div>
+    </div></div>      
+`;
```

Comment:
- [ ] Instead of putting the project details directly in the HTML, Please use a JavaScript array to store all of the information for all projects and insert them in HMTL elements using `${}` for all projects.
![image](https://user-images.githubusercontent.com/83803414/195178466-4bee9784-ead4-4993-a1d7-4bae35aba9ea.png)